lab_5_scrapper/scrapper.py

```diff
@@ -204,12 +344,26 @@ def prepare_environment(base_path: Union[pathlib.Path, str]) -> None:
     Args:
         base_path (Union[pathlib.Path, str]): Path where articles stores
     """
+    if not base_path.exists():
+        base_path.mkdir()
+    base_path.rmdir()
```

Comment:
if it is a directory with content, then it will not be so easy

Reply:
changed it

Reply:
fixed it